templates/review-form.html

```diff
@@ -18,7 +18,11 @@ <h2 class="text-center">Leave a Review</h2>
                 {{ field(class='form-control') }}
                 {% endif %}
             {% endfor %}
-            <button class="btn btn-primary my-2">Save changes!</button>
+            <div id="uploadProgressContainer" style="display: none;">
```

Comment:
nit: avoid inline styles and create a class

Reply:
updated to use bootstrap class